pr-data.csv

```diff
@@ -5907,6 +5907,9 @@ https://github.com/nationalsecurityagency/timely,3a8cbd3378cffdc96aad1c52e9b0b2e
 https://github.com/nationalsecurityagency/timely,3a8cbd3378cffdc96aad1c52e9b0b2e1481ca45f,server,timely.store.iterators.TimeSeriesGroupingIteratorTest.testMultipleTimeSeriesMovingAverage,NDOI,MovedOrRenamed,,https://github.com/NationalSecurityAgency/timely/commits/18eecfd7b08abe3eb1830b1118e687493ba0f739
 https://github.com/nationalsecurityagency/timely,3a8cbd3378cffdc96aad1c52e9b0b2e1481ca45f,server,timely.store.iterators.TimeSeriesGroupingIteratorTest.testTimeSeriesDropOff,NDOI,MovedOrRenamed,,https://github.com/NationalSecurityAgency/timely/commits/18eecfd7b08abe3eb1830b1118e687493ba0f739
 https://github.com/neo4j/neo4j,c68156edf24164435ab1ac257ec633134c2887f7,community/lock,org.neo4j.kernel.impl.locking.forseti.LockingCompatibilityTest$RWLock.shouldIncludeDeadlockCycleForSimpleUpdateDeadlock,OD,Opened,https://github.com/neo4j/neo4j/pull/13702,https://github.com/neo4j/neo4j/issues/13701
+https://github.com/netty/netty,0a3df28b2d8ed7b2f4f5389c81b2306c055e4762,codec-http,io.netty.handler.codec.http.websocketx.extensions.WebSocketExtensionUtilTest.computeMergeExtensionsHeaderValueWhenConflictingUserDefinedHeader,ID,Opened,https://github.com/netty/netty/pull/15862,Test fails with nondex seed 974622.
```

Comment:
Remove the code about the seed, as it won't be reproducible after a while. Also squash all your commits into 1.